.github/pull_request_template.md

```diff
@@ -1,28 +1,18 @@
-<!-- These comments are hidden when you submit the PR,
-so you do not need to remove them!
-
-**Note: If this PR closes a JIRA ticket, make sure the title
+<!-- If this PR closes a JIRA ticket, make sure the title
 starts with the JIRA issue number, for example
-JP-123: <Fix a bug>
-**
+JP-1234: <Fix a bug>
 -->
 
 Closes #
-
 Resolves JP-
```

Comment:
Please extend the template to include the proper formatting for an actual link:
`Resolves [JP-nnnn](https://jira.stsci.edu/browse/JP-nnnn)`

Reply:
Ah, great idea.  Done.